lib/logstash/inputs/LogStashRecordProcessorFactory.rb

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+#
+#Copyright 2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+#
+require 'java'
+require_relative "LogStashRecordProcessor"
+
+require "logstash-input-dynamodb_jars"
+
+module KCL
+  include_package "com.amazonaws.services.kinesis.clientlibrary.interfaces"
+end
+
+module Logstash
+  module Inputs
+    module DynamoDB
+      class LogStashRecordProcessorFactory
```

Comment:
I think here we can apply the same recommendation as earlier. The Logstash part in the class name looks a bit redundant to me :smile: